src/adapters/PeopleAdapter.js

```diff
@@ -6,11 +6,26 @@ import {throwError} from 'rxjs';
  * adhering to the exact parameters and structure of the returned objects
  */
 export default class PeopleAdapter {
+  /**
+   * A Person object with details about the person
```

Comment:
Would it be too specific to say:
`A Person object with details about a Webex Teams user`.

Reply:
I think it might be too specific. Users of the library could populate the data from an external source. I know it isn't very likely, but might as well keep it generic enough for those use cases.

Reply:
Sure